nilearn/surface/tests/_testing.py

```diff
@@ -1,35 +0,0 @@
-"""Helper functions for testing surface code."""
-
-import numpy as np
-from scipy.spatial import Delaunay
-
-from nilearn.conftest import _rng
-from nilearn.surface import InMemoryMesh
-
-
-def generate_surf():
-    """
-    Generate random surface object for testing input output functions.
-
-    This does not generate meaningful surfaces.
-    """
-    rng = _rng()
-    coords = rng.random((20, 3))
-    faces = rng.integers(coords.shape[0], size=(30, 3))
-    return InMemoryMesh(coordinates=coords, faces=faces)
-
-
-def flat_mesh(x_s, y_s, z=0):
```

Comment:
I think we should keep this fixture because:

- it creates "meaningful" meshes. I haven't tried plotting the random "not-meaningful" one vs. this one but I think "meaningful" here means that the edges are contiguous and form one continuous surface.

- the "meaningful" ones we have right now are quite small (only 9 vertices). With this one, we can create larger ones that might be useful in some places for example in #4756